src/packages/solid/components/CardContent/CardContent.tsx

```diff
@@ -0,0 +1,179 @@
+   *
+   * component will be shown/hidden on focus unless  is false
+   *
+   * states are not forwarded to this component. if it should react to states
+   * applied to the CardContent, they must also be set on this component
+   */
+  topRight?: NodeProps['children'];
+
+  /**
+   * component rendered to center of the CardContent, flex-aligned to the bottom border
+   *
+   * component will be shown/hidden on focus unless  is false
+   *
+   * states are not forwarded to this component. if it should react to states
+   * applied to the CardContent, they must also be set on this component
+   */
+  inset?: NodeProps['children'];
+
+  /**
+   * component rendered below the CardContent
+   *
+   * component will be shown/hidden on focus unless shouldCollapse is false
+   *
+   * states are not forwarded to this component. if it should react to states
+   * applied to the CardContent, they must also be set on this component
+   */
+  bottom?: NodeProps['children'];
+
+  /**
+   * props passed to the child ProgressBar, rendered at the bottom of the component.
+   * ProgressBar will not appear if progressBar.progress is 0/falsy
+   */
+  progressBar?: Partial<ProgressBarProps> | undefined;
+
+  /**
+   * component rendered in right section of the CardContent on the top
+   *
+   * component will be shown/hidden on focus if shouldCollapse is true
+   *
+   * states are not forwarded to this component. if it should react to states
+   * applied to the CardContent, they must also be set on this component
+   */
+  metadataTop?: NodeProps['children'];
+
+  /**
+   * component rendered in right section of the CardContent on the bottom
+   *
+   * component will be shown/hidden on focus if shouldCollapse is true
+   *
+   * states are not forwarded to this component. if it should react to states
+   * applied to the CardContent, they must also be set on this component
+   */
+  metadataBottom?: NodeProps['children'];
+  /**
+   * sets the component's color palette
+   */
+  tone?: Tone;
+
+  padding?: number[];
+
+  style?: Partial<CardContentStyles>;
+}
+
+const CardContent: Component<CardContentProps> = props => {
+  const [isFocused, setIsFocused] = createSignal(false);
+
+  const formatTile = (focusState: boolean, shouldCollapse: boolean) => {
+    return shouldCollapse && !focusState;
+  };
+  const collapsedTile = createMemo(() => formatTile(isFocused(), props.shouldCollapse ?? false));
+
+  return (
+    <node
+      use:withPadding={
+        props.padding ??
+        styles.Container.tones[props.tone ?? styles.tone]?.padding ??
+        styles.Container.base.padding
+      }
+      {...props}
+      style={[
+        ...[props.style].flat(),
+        styles.Container.tones[props.tone ?? styles.tone],
+        styles.Container.base
+      ]}
+      onFocus={() => setIsFocused(true)}
+      onBlur={() => setIsFocused(false)}
+      states={{ collapsed: isFocused(), focus: isFocused() }}
```

Comment:
is the `focus: isFocused()` necessary? I thought focus would automatically get applied